tests/unittests/TagServiceTest.php

```diff
@@ -7,23 +7,32 @@
 use OCP\SystemTag\ISystemTag;
 use OCP\SystemTag\ISystemTagManager;
 use OCP\SystemTag\ISystemTagObjectMapper;
+use PHPUnit\Framework\MockObject\Exception;
```

Comment:
This for example is where I really don't like the doc comments. The import is now just for the comment? It is not used in any code-part and the functions here are test functions, that will never be called by any developer from the outside.

Also the php autocompleters will already show you the signature of the function, so you will see the informations already in your IDE.
![image](https://github.com/GDATASoftwareAG/nextcloud-gdata-antivirus/assets/13208176/09780aca-1fb1-4724-b46c-df7c61e44a8a)

The doc-comment is just adding the throws but they are not even complete.
![image](https://github.com/GDATASoftwareAG/nextcloud-gdata-antivirus/assets/13208176/bee28d33-4102-4093-89d7-27c81c4c1336)

The InvalidArgumentException exception for example is from the global namespace.